warehouse/deltalake/deltalake.go

```diff
@@ -255,23 +252,33 @@ func Connect(cred *databricks.CredentialsT, connectTimeout time.Duration) (dbHan
 		return
 	}
 
-	dbHandleT = &databricks.DBHandleT{
+	deltalakeClient = &deltalakeclient.DeltalakeClient{
+		Logger:         dl.Logger,
 		CredConfig:     connConfig,
 		CredIdentifier: identifier,
 		Conn:           conn,
 		Client:         dbClient,
 		Context:        ctx,
 	}
+
+	// Setting up catalog at the client level
+	if catalog := warehouseutils.GetConfigValue(Catalog, dl.Warehouse); catalog != "" {
+		sqlStatement := fmt.Sprintf("USE CATALOG `%s`;", catalog)
```

Comment:
Didn't get what do you mean by sanitize?

Reply:
Can we use $1 here for capturing the input ?

Reply:
Currently, we don't have the support for this. Since we are delegating the queries to pyodbc driver, using a python client. Python client doesn't have the support this.

Reply:
I have added a [task](https://github.com/rudderlabs/rudder-server/pull/2878#discussion_r1082279038) in the backlog for tracking this.

Reply:
Sounds good 👍🏼